templates/expo-template-bare-minimum/ios/HelloWorld.xcodeproj/project.pbxproj

```diff
@@ -236,6 +238,25 @@
 			shellScript = "diff \"${PODS_PODFILE_DIR_PATH}/Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [ $? != 0 ] ; then\n    # print error to STDERR\n    echo \"error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\" >&2\n    exit 1\nfi\n# This output is used by Xcode 'outputs' to avoid re-running this script phase.\necho \"SUCCESS\" > \"${SCRIPT_OUTPUT_FILE_0}\"\n";
 			showEnvVarsInLog = 0;
 		};
+		353F5C274CB4DD6AD9FBEA89 /* [Expo] Configure project */ = {
+			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputFileListPaths = (
+			);
+			inputPaths = (
+			);
+			name = "[Expo] Configure project";
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "# This script configures Expo modules and generates the modules provider file.\nbash -l -c \"./Pods/Target\\ Support\\ Files/Pods-HelloWorld/expo-configure-project.sh\"\n";
```

Comment:
```suggestion
			shellScript = "# This script configures Expo modules and generates the modules provider file.\nbash -l -c \"${PODS_ROOT}/Target Support Files/Pods-HelloWorld/expo-configure-project.sh\"\n";
```

1. use `PODS_ROOT`
2. since it has double quotes already, maybe we don't need to escape the spacings.

Reply:
I think I'll need to change it in autolinking first, so I'll do that in a separate PR

Reply:
So the code that generates this is here:
https://github.com/expo/expo/blob/d7ad395fe77f9071c27b6baf7e6df4ce706663fc/packages/expo-modules-autolinking/scripts/ios/project_integrator.rb#L204-L211

Given `script_relative_path` is constructed in the second line here:
https://github.com/expo/expo/blob/d7ad395fe77f9071c27b6baf7e6df4ce706663fc/packages/expo-modules-autolinking/scripts/ios/project_integrator.rb#L181-L182

So it already points to Cocoapods `target.support_files_dir`.
Do you think we should change it to `${PODS_ROOT}/Target Support Files`?